workspaces/github-actions/packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -0,0 +1,367 @@
+import React from 'react';
+import { Button, Grid } from '@material-ui/core';
+import {
```

Comment:
Because this is just focusing on GitHub actions it might make sense to trim these catalog pages down even further and removing api docs and the catalog-graph.